ArduPlane/Plane.h

```diff
@@ -937,6 +937,15 @@ class Plane : public AP_Vehicle {
     bool verify_command_callback(const AP_Mission::Mission_Command& cmd);
     float get_wp_radius() const;
 
+    void do_nav_delay(const AP_Mission::Mission_Command& cmd);
+    bool verify_nav_delay(const AP_Mission::Mission_Command& cmd);
+
+    // Delay the next navigation command
+    struct {
+        uint32_t time_max_ms;
+        uint32_t time_start_ms;
+    } nav_delay;
```

Comment:
the use of a structure is a small but nice improvement on the implementation in the other vehicles.